guides/source/active_record_postgresql.md

```diff
@@ -242,10 +242,11 @@ article.save!
 
 ### UUID
 
-* [type definition](http://www.postgresql.org/docs/9.3/static/datatype-uuid.html)
-* [generator functions](http://www.postgresql.org/docs/9.3/static/uuid-ossp.html)
+* [type definition](http://www.postgresql.org/docs/9.4/static/datatype-uuid.html)
+* [pgcrypto generator function](http://www.postgresql.org/docs/9.4/static/pgcrypto.html#AEN159361)
+* [uuid-ossp generator functions](http://www.postgresql.org/docs/9.4/static/uuid-ossp.html)
 
-NOTE: you need to enable the `uuid-ossp` extension to use uuid.
+NOTE: you need to enable the `pgcrypto` or `uuid-ossp` extension to use uuid.
```

Comment:
let's mention that `gen_random_uuid` from `pgcrypto` only exists in PostgreSQL >= `9.4`